clang/include/clang/Basic/DiagnosticLexKinds.td

```diff
@@ -361,7 +361,11 @@ class NonportablePath  : Warning<
   "non-portable path to file '%0'; specified path differs in case from file"
   " name on disk">;
 def pp_nonportable_path : NonportablePath,
-  InGroup<DiagGroup<"nonportable-include-path">>;
+  InGroup<NonportablePath>;
```

Comment:
I would be nice to give the diagnostic group a different identifier (the name we expose to users is fine!) so that we don't have this weird naming conflict between the diagnostic group and the `class NonportablePath` definition a few lines up from here.